pkg/bmc/interface.go

```diff
@@ -0,0 +1,131 @@
+// Copyright 2018 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package bmc
+
+import (
+	"fmt"
+	"log"
+	"math/rand"
+	"time"
+
+	"github.com/vishvananda/netlink"
+	"golang.org/x/sys/unix"
+	"github.com/insomniacslk/dhcp/dhcpv6"
```

Comment:
Did you look at u-root's dhcp6client?

Reply:
I did not! I assumed u-root didn't have a dhcp6client. I'll compare them, but it looks pretty nice from initial scan. Thanks for flagging.